tests/functional/remote/06-poll.t

```diff
@@ -53,7 +53,7 @@ log_scan \
     10 \
     1 \
     '\[foo\.1 submitted .* (polled)foo' \
-    '\[foo\.1 succeeded .* (polled)succeeded'
+    '\[foo\.1 .* (polled)succeeded'
```

Comment:
Locally, it polls again after already discovering the task succeeded
```
10:33:14 INFO - [foo.1 submitted job:01 flows:1] (polled)foo at 2021-11-02T10:33:10Z
10:33:14 INFO - [foo.1 submitted job:01 flows:1] (polled)succeeded at 2021-11-02T10:33:11Z
...
10:33:15 INFO - [foo.1 succeeded job:01 flows:1] (polled)foo at 2021-11-02T10:33:10Z
10:33:15 DEBUG - [foo.1 succeeded job:01 flows:1] unhandled: foo
10:33:15 INFO - [foo.1 succeeded job:01 flows:1] (polled)succeeded at 2021-11-02T10:33:11Z
```
With the lower polling rate on GH Actions, it doesn't poll again after discovering it succeeded, so we only get
```
10:33:14 INFO - [foo.1 submitted job:01 flows:1] (polled)foo at 2021-11-02T10:33:10Z
10:33:14 INFO - [foo.1 submitted job:01 flows:1] (polled)succeeded at 2021-11-02T10:33:11Z
```

Reply:
Not sure why this was testing "succeeded -> succeeded", I think that can only happen if the polling interval is similar to the main loop interval (which it was) which would explain why this test was flaky with high CPU load!

Suggest swapping "succeeded" for "submitted" so that this is still testing that polling can achieve the required state changes.

```
'\[foo\.1 submitted .* (polled)succeeded'
```

> **Note:** The `log-scan` thingy does not require messages to be adjacent, it only checks that they are present and in the right order.